packages/scss/src/_typo.scss

```diff
@@ -7,39 +7,39 @@ html {
 }
 
 body {
-	background: $greyBackground;
-	color: $grey;
+	background: #F5F5F5;
+	color: _get($luTheme, "palette.colors.greyStandard");
```

Comment:
As you repeat `_get($luTheme, "path")` everywhere, a `_theme("path")` shorthand function could be useful.